src/main/webapp/app/overview/student-questions/student-questions.scss

```diff
@@ -1,15 +1,108 @@
 @import 'src/main/webapp/content/scss/artemis-variables';
 
-.add-action {
-    color: $primary;
+$background-color: #353d47;
+
+.questions-container {
+    display: flex;
+    height: 100%;
+}
+
+.expanded-questions {
+    width: 30vw;
+
+    .draggable-left {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        min-width: 15px;
+    }
+
+    .card {
+        width: inherit;
+        min-width: 200px;
+
+        .card-header {
+            display: inline-flex;
+            justify-content: space-between;
+            align-items: center;
+            background-color: $background-color;
+            cursor: pointer;
+
+            .card-title {
+                display: flex;
+            }
+        }
+    }
+}
+
+.collapsed-questions {
+    width: 50px;
+    justify-content: space-between;
+    flex-flow: column nowrap;
+    background-color: $background-color;
+    margin-left: 15px;
+    cursor: pointer;
+
+    span {
+        writing-mode: vertical-rl;
+        margin: auto;
+    }
+
+    fa-icon {
+        display: flex;
+        justify-content: center;
+        padding: 15px 0;
+    }
 }
 
-.question-subtitle {
+.question-subtitle,
+.answer-subtitle {
     font-size: 12px;
 }
 
-.answer-approved {
-    color: green;
-    background-color: #f2f9f2;
+.question-text,
+.answer-text {
+    background: rgba(0, 0, 0, 0.1);
+    border-radius: 3px;
```

Comment:
```suggestion
    border-radius: 3px;
    
    ::ng-deep p {
        margin: 0;
    }
```

Reply:
I implemented that as well in my last commit. 
And thanks for all the suggestions btw 😄